app/views/users/posts.html.erb

```diff
@@ -0,0 +1 @@
+Here is the user <%= params[:id] %> posts list
```

Comment:
- [ ] Kindly make sure to store this file in # app/views/posts/index.html.erb and rename this file as index.html.erb